Makefile

```diff
@@ -23,6 +23,22 @@ help: ## display help for this makefile
 rm-pycache: ## remove all __pycache__ files (run if encountering issues with pycharm debugger (containers exiting prematurely))
 	find . -name '__pycache__' | xargs rm -rf
 
+### Connecting to remote dbs #########################################
+remotedb: # Get a psql console on a remote db (from OSX only!)
```

Comment:
can we rename this to `create-remotedb-uri`?

Reply:
It doesn't just create a URI - it passes the URI to psql and actually connects to the db. Should we include psql in the target name?

Reply:
nevermind, see other comment.